content/team/team.json

```diff
@@ -1,58 +1,52 @@
-      "linkedin": "https://www.linkedin.com/in/hiimchrislim"
-    },
     {
       "name": "Jessica Anderson",
-      "imgSrc": "/team/jess.jpg",
-      "position": "Operations Director",
+      "imgSrc": "/team/2021/jess.jpg",
+      "position": "Executive Director",
       "message": "Jessica was accepted into UTM as an Accounting Student, but ended up wanting to do Computer Science, so she is ready to tackle any of your POSt questions. She always struggles to choose which courses are most important to take and she's ready to share her findings!",
       "discord": "janders#8886",
       "email": "jess.anderson@mail.utoronto.ca",
       "linkedin": "https://www.linkedin.com/in/jessicaanderson9"
     },
-    {
-      "name": "VyVy Ngo",
-      "imgSrc": "https://avatars.githubusercontent.com/u/45834118?s=460&u=83382baac5f0a500fa99646a4d5899fe391f7328&v=4",
-      "position": "Student Resource Curator",
-      "message": "As the student resource curator, VyVy wants to create a welcoming computer science environment by providing students access to any information they need. She has a passion learning and grabs any information within her reach. In her spare time, she enjoys knitting and some Skyrim.",
-      "discord": "SleepyVyvy#9713",
-      "email": "vyvy.ngo@mail.utoronto.ca",
-      "linkedin": "https://www.linkedin.com/in/tuong-vy-n-614b4213b/"
-    },
     {
       "name": "Nameera Azim",
-      "imgSrc": "/team/Nameera.png",
-      "position": "Student Culture Coordinator",
-      "message": "Nameera dedicates her time to connecting with the students within the Computer Science community to learn about what type of support they need and advocating for changes that improve their University experience. Feel free to shoot her message if you want to talk about anything - from internships to Earnest Hemmingway to new Marvel movies.",
+      "imgSrc": "/team/2021/Nameera.png",
+      "position": "Director of Communications",
+      "message": "Nameera is a third-year student specializing in Computer Science and minoring in both Statistics and Business. On lazy days, you can probably find her reading, painting, fangirling over an MCU movie, watching the Premier League, or messing around with her ukulele! Feel free to reach out!",
       "discord": "Nameera#2802",
       "email": "nameera.azim@mail.utoronto.ca",
       "linkedin": "https://www.linkedin.com/in/nameera-azim-1924a21b6/"
     },
+    {
+      "name": "Naaz Sibia",
+      "imgSrc": "/team/2021/Naaz.png",
+      "position": "Director of Operations",
+      "message": "Naaz likes preference learning and Human-Computer Interaction (with a CS-ed focus). She also likes talking about mental health, and courses she's TA'ed. When not looking into the mentioned topics, or working to finish her bachelor's degree, she likes having 3 am discussions with friends, drawing chibis, and eating Chinese food.",
+      "discord": "Blossom#1234",
+      "email": "vyvy.ngo@mail.utoronto.ca",
```

Comment:
Might want to change this to Naaz' email and Linkedin 😅

Reply:
Thanks haha, can’t believe I missed that